.gitignore

```diff
@@ -1,50 +1,50 @@
-# Compile products
-*.pyc
-*.class
-*.crc
-*.o
-
-# SBT, Maven specific
-.cache
-.history
-.lib/
-lib_managed/
-src_managed/
-project/boot/
-project/plugins/project/
-dependency-reduced-pom.xml
-
-# IDE specific
-.scala_dependencies
-.worksheet
-*.iml
-.idea/
-.vscode/
-
-# macOS specific
-.DS_Store
-
-# data files
-model*.[0-9]*
-state*.[0-9]*
-
-# other
-nohup.out
-*.log
-*.lock
-*.un~
-*.idx
-.ipynb_checkpoints/
-.project
-.settings/
-*/.cache-main
-*/.cache-tests
-*/.classpath
-*/.project
-*/.settings/
-*.so
-__pycache__
-*.egg-info
-target
-build
-dist
+# Compile products
+*.pyc
+*.class
+*.crc
+*.o
+
+# SBT, Maven specific
+.cache
+.history
+.lib/
+lib_managed/
+src_managed/
+project/boot/
+project/plugins/project/
+dependency-reduced-pom.xml
+
+# IDE specific
+.scala_dependencies
+.worksheet
+*.iml
+.idea/
+.vscode/
+
+# macOS specific
+.DS_Store
+
+# data files
+model*.[0-9]*
+state*.[0-9]*
+
+# other
+nohup.out
+*.log
+*.lock
+*.un~
+*.idx
+.ipynb_checkpoints/
+.project
+.settings/
+*/.cache-main
+*/.cache-tests
+*/.classpath
+*/.project
+*/.settings/
+*.so
+__pycache__
+*.egg-info
+target
+build
+dist

```

Comment:
I am not sure what's happening to this file. There seems to be no changes but only whitespace(invisible ones). To make sure nothing is broken, we should fix this problem before merging the PR.